docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to add a new user.
+2. Under **Settings**, click **Access**.
+1. Click **+ Add**
+1. Add the user's details and choose their permissions.
+4. Click **Save**.
+5. After you add a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to create new users.
 
-You can either grant the `Project admin` role to the user, which will give them `Admin` access to every environment in the project, or grant specific permissions for each environment type.
+* `platform user:add`
 
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
+Say you want to add `user1@example.com` to the project with **Project Admin** permissions:
 
-In order to push and pull code (or to SSH into one of the project's environments), the user needs to add an SSH key or use the Platform CLI.
+```bash
+platform user:add user1@example.com -r admin
+```
+Once you add the user to the project, they receive an invitation email to join the project.
 
-If the user already has an account, they receive an email with a link to access the project.
+Use `platform list` to get the full list of commands.
 
-## Manage users permissions with the CLI
+{{< /codetabs >}}
 
-You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md). to fully manage your users and integrate with any automated system.
+Once you add a user to a project, they receive an invitation to join the project.
 
-Available commands:
+## Manage existing users
 
-* `platform user:add`
-  * Add a user to the project
-* `platform user:delete`
-  * Delete a user
-* `platform user:list` (`users`)
-  * List project users
-* `platform user:role`
-  * View or change a user's role
+### Delete a user from a project
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+To delete a user from a project, follow these steps:
 
-```bash
-platform user:add alice@example.com -r admin
-```
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to delete a user.
+2. Under **Settings**, click **Access**.
+3. Select the user you want to delete and click **Delete**
+4. Click **Save**.
+5. After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to delete existing users.
+
+* `platform user:delete`
```

Comment:
```suggestion
`platform user:delete`
```
I don't see a reason for this to be a bulleted list (there aren't other items in it. It might also help to have an introductory phrase.